compiler/lib/compiler.go

```diff
@@ -271,3 +277,41 @@ func (c *Compiler) GetLoader() *starlarkLoader {
 		moduleService: c.ModuleService,
 	}
 }
+
+func newConfigRollout(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
+	var config starlark.Value
+
+	starlark.UnpackArgs(fn.Name(), args, kwargs, "value", &config)
```

Comment:
check for error

Reply:
I've added some comments in regard to why the error check was skipped and added a different check functionality.